app/src/test/java/mozilla/lockbox/store/TelemetryStoreTest.kt

```diff
@@ -64,8 +65,8 @@ class TelemetryStoreTest : DisposingTest() {
     fun testApplyConfig() {
         val applyObserver = createTestObserver<Context>()
         wrapper.applySubject.subscribe(applyObserver)
-        subject.injectContext(RuntimeEnvironment.application)
-        applyObserver.assertValue(RuntimeEnvironment.application)
+        subject.injectContext(ApplicationProvider.getApplicationContext())
```

Comment:
`ApplicationProvider.getApplicationContext()` can be pulled out into a `context` val.